### ISSUE TITLE
User Portfolio Graph Component

### ISSUE THREAD

**Issue by Eric-Hong99** [Drem-Labs/drem-interface#10]
Implement the user portfolio graph component. A portfolio graph includes the following:
- Balance Label
- A visualization graph
- Total balance 
- Percentage increase within a given time
- Buttons for different time periods 

Personally recommend using d3.js to create a graph within the component with dataset(s) but can be discussed 

DOD: Make a sample portfolio graph component that displays relevant information/graph(s) based on which time period button is currently clicked + is visually consistent with the image below. 

![image](https://user-images.githubusercontent.com/34068439/180443306-2df09586-b55f-480f-8055-f2ae07a833de.png)

**Comment by PatrickStarLee** [comment]
Hey guys. I just wanted to get your guy's thoughts on how we should reuse our graph component. Right now, I'm reusing a graph component that I made for the crypto graph and this is what it looks like. 
![image](https://user-images.githubusercontent.com/64600761/194672044-3c33f67b-8492-43c0-a695-8e94ce794e13.png)

I made this border of what the original prototype looks like with just the card component.
![image](https://user-images.githubusercontent.com/64600761/194672169-3ecc7fe2-53b8-4709-b967-af6dc77704cb.png)

@Evan-Nishi @Radilx @lpolt Thoughts?

**Comment by Evan-Nishi** [comment]
I agree, we should just reuse the graph component and remove the logo of the currency.

**Comment by PatrickStarLee** [comment]
![image](https://user-images.githubusercontent.com/64600761/194726426-3908ad0f-a9b4-4820-b0c2-8d6f3ae1d192.png)

Here's the final product that I got
